ruby/extractor/src/diagnostics.rs

```diff
@@ -92,7 +93,7 @@ impl LogWriter {
         DiagnosticMessage {
             timestamp: chrono::Utc::now(),
             source: Source {
-                id: format!("{}/{}", self.extractor, id),
+                id: format!("rb/{}", id),
```

Comment:
The Ruby extractor is actually language agnostic, so having `rb` in this file is not great. I'll refactor this later.